### PR TITLE
 Functional tests for Cincinnati operator

### PR DESCRIPTION
**Pull request by pawanpinjarkar**

### CODE REVIEW

File: ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml [hunk 1]
```diff
@@ -35,6 +35,10 @@ tests:
   commands: make unit-test
   container:
     from: src
+- as: functional
+  commands: make functest
+  container:
+    from: src
```
Comment (rwsu) [inline ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml:41]: The functional tests will execute against a running OpenShift cluster (https://github.com/openshift/release/blob/master/ci-operator/README.md) . You'll need to change this configuration to:

- as: cincinnati-operator-e2e-aws
  commands: |
    make functest
  openshift_installer_src:
    cluster_profile: aws

Comment (rwsu) [reply ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml:41]: And then regenerate the jobs using "make jobs" and check in the changes under ci-operator/jobs/openshift/cincinnati-operator/.

Comment (pawanpinjarkar) [reply ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml]: Done.

File: ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml [hunk 2]
```diff
@@ -31,10 +31,14 @@ tag_specification:
...
-- as: unit
-  commands: make unit-test
-  container:
-    from: src
+  - as: unit
+    commands: make unit-test
+    container:
+      from: src
+  - as: cincinnati-operator-e2e-aws
+    commands: make functest
```
Comment (sacharya) [inline ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml:39]: s/functest/func-test/

Comment (pawanpinjarkar) [reply ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml]: Thank you, Done

File: ci-operator/jobs/openshift/cincinnati-operator/openshift-cincinnati-operator-master-presubmits.yaml [hunk 3]
```diff
@@ -1,5 +1,88 @@
...
+        - --template=/usr/local/cincinnati-operator-e2e-aws
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: JOB_NAME_SAFE
+          value: cincinnati-operator-e2e-aws
+        - name: TEST_COMMAND
+          value: make functest
```
Comment (sacharya) [inline ci-operator/jobs/openshift/cincinnati-operator/openshift-cincinnati-operator-master-presubmits.yaml:38]: Run `make jobs` again.

Comment (pawanpinjarkar) [reply ci-operator/jobs/openshift/cincinnati-operator/openshift-cincinnati-operator-master-presubmits.yaml]: Done

**Comment by pawanpinjarkar** [comment]
Accidently this PR was closed. New PR is here https://github.com/openshift/release/pull/9161.

**Pull request closed without merging**